src/mlpack/core/optimizers/parallel_sgd/decay_policies/constant_step.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file constant_step.hpp
+ * @author Shikhar Bhardwaj
+ *
+ * Constant step size policy for parallel Stochastic Gradient Descent.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_PARALLEL_SGD_CONSTANT_STEP_HPP
+#define MLPACK_CORE_OPTIMIZERS_PARALLEL_SGD_CONSTANT_STEP_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Implementation of the ConstantStep stepsize decay policy for parallel SGD.
+ */
+class ConstantStep
+{
+ public:
+  ConstantStep(double initalStep) : step(initalStep) { /* Nothing to do */ }
+
+  /**
+   * This function is called in each iteration before the gradient update.
+   *
+   * @param n_epoch The iteration number for which the stepsize is to be
```

Comment:
Can you use camel casing for all names, I know the parameter isn't used, but doxygen should pick it up anyway.